modules/extensions/src/data-filter/shader-module.ts

```diff
@@ -55,6 +59,29 @@ void dataFilter_setValue(DATAFILTER_TYPE valueFromMin, DATAFILTER_TYPE valueFrom
         step(filter_min, valueFromMin) * step(valueFromMax, filter_max)
       );
     }
+
+    #if DATACATEGORY_CHANNELS == 1 // One 128-bit mask
```

Comment:
Nit: I am a bit wary about #defines do create some maintenance issues as not all source code is parsed unless the test suite is quite comprehensive, so syntax errors can creep in. But it might be worth it here.